source/reference/method/MongoDBCollection-find.txt

```diff
@@ -52,9 +52,15 @@ Parameters
 
      * - batchSize
        - integer
-       - The number of documents to return in the first batch. Defaults to
-         ``101``. A batchSize of ``0`` means that the cursor will be
-         established, but no documents will be returned in the first batch.
+       - The maximum number of documents to return in the first batch. By default, the ``find()`` 
+         command has an initial batch size of ``101`` documents. This
+         option can enforce a smaller limit than 16 mebibytes, but not a larger
+         one. If you set ``batchSize`` to a limit that results in batches larger than
+         16 MiB, this option has no effect.
+
+         A batchSize of ``0`` means that the cursor will be established, but no documents 
+         will be returned in the first batch. This may be useful for quickly returning a cursor
+         or failure from ``aggregate`` without doing significant server-side work.
```

Comment:
I: remove ``aggregate`` from this description as its about the find method. I think JM meant that setting this option is useful for aggregation contexts but that users might use it for finds too